lib/Remote/RemoteWebElement.php

```diff
@@ -325,17 +326,33 @@ public function sendKeys($value)
     {
         $local_file = $this->fileDetector->getLocalFile($value);
         if ($local_file === null) {
-            $params = [
-                'value' => WebDriverKeys::encode($value),
-                ':id' => $this->id,
-            ];
+            if ($this->w3cCompliant) {
```

Comment:
Could this be moved to the compatibility layer as well?

Reply:
Isn't it better to keep it here. If at some point we remove support for the legacy protocol, it will be easier this way.

Reply:
As it something we want to do (#657), I suggest to leave this as is.